kite-morphlines/kite-morphlines-hadoop-rcfile/src/test/java/org/kitesdk/morphline/hadoop/rcfile/ReadRCFileTest.java

```diff
@@ -135,7 +153,12 @@ private void createRCFile(final String fileName, final int numRecords,
       BytesRefArrayWritable dataWrite = new BytesRefArrayWritable(maxColumns);
       dataWrite.resetValid(maxColumns);
       for (int column = 0; column < maxColumns; column++) {
-        Text sampleText = new Text("ROW-NUM:" + row + ", COLUMN-NUM:" + column);
+        Writable sampleText = new Text(
+            "ROW-NUM:" + row + ", COLUMN-NUM:" + column);
+        // Set the last column of the last row as null
```

Comment:
Is only setting the last element to null sufficient testing? What about null somewhere in the middle? What if the first element is null?